tb/emerge.py

```diff
@@ -0,0 +1,48 @@
+import tb.yml as tbyml
+import os
+
+
+tbyml.read_yml()
+yml = tbyml.get_yml()
+
+
+EMERGE_COMMAND = """
+emerge \
+--autounmask \
+--autounmask-write=n \
+--buildpkg=y \
+--buildpkg-exclude="{bin_exclude}" \
+--changed-deps=y \
+--color=y \
+--complete-graph=y \
+--config-root="{config_root}" \
+--deep \
+--exclude="{exclude}" \
+--getbinpkg=n \
+--getbinpkgonly=n \
+--jobs="{jobs}" \
+--load-average="{jobs}" \
+--misspell-suggestions=n \
+--newuse \
+--root="{root}" \
+--update \
+--with-bdeps=y \
+""".strip()
+
+
+def eformat(bin_exclude=yml["BINEXCLUDE"],
+        exclude=yml["EXCLUDE"],
+        target="/",
+        jobs=yml["JOBS"],
+        root=yml["ROOTSDIR"],
```

Comment:
`root` is unused now that we use `target` to set the root directory to the emerge command below.

Reply:
fixed!